applications/msai.md

```diff
@@ -0,0 +1,290 @@
+|    0b. | Documentation                          | The extended API will be documented                                                                                                                                                                                                                                                                                                      |
+|    0c. | Testing Guide                          | Each epic describes testing opportunities.  [ 1. Create a health blockchain](https://trello.com/c/LZ3c4K3T/3-create-basic-parachain-with-useful-api) 2. Create openapi specifcations for reading and writing data to and from the NFT smart contract (Depends on blockchain nft tech used) .                                             |
+|    0d. | Docker                                 | A dockerized setup of the parachain will be provided after several iterations when we have a working secure application against a customised subtrate blockchain with extended off chain worker local storage                                                                                                                            |
+|    0e. | Article                                | Press release will be prepared a few weeks before the official rollout of the network. The link to the article will be sent when ready                                                                                                                                                                                                   | 
+|    1a. | Node Repo                              | Complete the deployment of the basic public chain with example  storage of a native DB format through the off chain worker local storage API.                                                                                                                                                                                            |
+|    2a. | Create New Substrate Pallet            | [ 1. Create demo application](https://trello.com/c/LZ3c4K3T/3-create-basic-parachain-with-useful-api) New parachain with smart contract facility through creation of a dedicated Pallet                                                                                                                                                  |                                                                                                                                                        |
+|    3a. | Extend with Storage Pod API            | Basic data storage API (with OpenAPI specifications) built on top of OFW adding some of the functionality of Solid pods                                                                                                                                                                                                                  |
```

Comment:
What's "OFW"? Off-Chain Worker?
What's "some of the functionality"? Specify further please.

Reply:
YEs, I expanded on that. Basically it is to provide data redundancy. When storing data off chain, what if one node is offline? Blockchain can handle that. so to make sure our app works we need a protocol to distribute a copy of the off chain data or a few nodes. Nodes are put into groups  and each group would have copies of the same data. Not everyone as then update will become too slow.